src/mappings/DAOTracker/mapping.ts

```diff
@@ -0,0 +1,127 @@
+import { Address, ipfs, store } from '@graphprotocol/graph-ts';
+import {
+  BlacklistDAO,
+  DAOTracker,
+  ResetDAO,
+  TrackDAO,
+} from '../../types/DAOTracker/DAOTracker';
+import {
+  BlacklistedDAO,
+  DAOTrackerContract,
+  ResetDAO as ResetDAOEntity,
+} from '../../types/schema';
+import {
+  Avatar_0_0_1_rc_31,
+  Controller_0_0_1_rc_31,
+  DAOToken_0_0_1_rc_31,
+  Reputation_0_0_1_rc_31,
+} from '../../types/templates';
+import { equalStrings } from '../../utils';
+
+export function getDAOTrackerContract(address: Address): DAOTrackerContract {
+  let daoTracker = DAOTrackerContract.load(address.toHex()) as DAOTrackerContract;
```

Comment:
does the DAOTrackerContract is used by any anyone ? is that really needed ?

Reply:
I think it's necessary for administrative debugging / checking. Also in the UI if a user tries to view a DAO that's been blacklisted, we'd want to be able to show the reason as to why it's been blacklisted from the UI.